plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java

```diff
@@ -257,4 +264,25 @@ protected static List<Column> toHiveColumns(List<NestedField> columns)
                         Optional.empty()))
                 .collect(toImmutableList());
     }
+
+    protected CommitStatus checkCommitStatus(String metadataLocation)
+    {
+        AtomicReference<CommitStatus> status = new AtomicReference<>(CommitStatus.UNKNOWN);
+
+        Tasks.foreach(metadataLocation)
+                .retry(3)
+                .suppressFailureWhenFinished()
+                .exponentialBackoff(100, 5000, 600000, 4.0)
```

Comment:
where do this values come from?
is it copied from iceberg library?

Reply:
values are taken from 

https://github.com/krvikash/trino/blob/68d3da6770f3195dec19a6812797210d8b4c4618/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java#L226